### PR TITLE
Oldie

### PR DESCRIPTION
**Pull request by renatoi**
@hankhsiao @thierryk

### CODE REVIEW

**Comment by yahoocla** [comment]
CLA is valid!

File: bin/atomizer [hunk 1]
```diff
@@ -13,18 +13,32 @@ process.title = 'atomizer';
...
+  .option('-c, --config [file]', 'source config file if any.')
+  .option('-o, --outfile [file]', 'destination config file.')
+  .option('-n, --namespace [namespace]', 'adds the given namespace to all generated Atomic CSS selectors.')
+  .option('-H, --helpersNamespace [namespace]', 'adds the given namespace to all helper selectors.')
+  .option('--rtl', 'swaps `start` and `end` keyword replacements with `right` and `left`.')
+  .option('--ie', 'adds old IE hacks to the output.')
+  .option('--verbose', 'show additional log info (warnings).')
+  .parse(process.argv);
+
+if (!process.argv.slice(2).length) {
```
Comment (hankhsiao) [inline bin/atomizer]: Why not just `process.argv.length <= 2`?

Comment (renatoi) [inline bin/atomizer]: Good call. I was using what `commander` had in their examples :) Fixed.

**Comment by hankhsiao** [comment]
:ship:

**Pull request merged**